controllers/twophase/types.go

```diff
@@ -136,41 +137,56 @@ func (r *Reconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 		r.Log.Info("Resuming")
 
 		dur := chaos.GetNextRecover().Sub(now)
-		if err := applyAction(ctx, r, req, dur, chaos); err != nil {
+		if startTime, err = applyAction(ctx, r, req, dur, chaos); err != nil {
 			return ctrl.Result{Requeue: true}, err
 		}
 
 	} else if chaos.GetNextStart().Before(now) {
-		nextStart, err := utils.NextTime(*chaos.GetScheduler(), now)
+		tempStart, err := utils.NextTime(*chaos.GetScheduler(), now)
 		if err != nil {
-			r.Log.Error(err, "failed to get next start time")
+			r.Log.Error(err, "failed to calculate the temp start time")
 			return ctrl.Result{}, err
 		}
 
-		nextRecover := now.Add(*duration)
-		if nextStart.Before(nextRecover) {
+		tempRecover := now.Add(*duration)
+		if tempStart.Before(tempRecover) {
 			err := fmt.Errorf("nextRecover shouldn't be later than nextStart")
-			r.Log.Error(err, "nextRecover is later than nextStart. Then recover can never be reached",
-				"nextRecover", nextRecover, "nextStart", nextStart)
+			r.Log.Error(err, "Then recover can never be reached. It may be because cron is greater than duration.")
```

Comment:
This sentence doesn't make sense to me. Can you be more specific here? It is not good to put `may be` in the error log.